lib/charms/tls_certificates_interface/v1/tls_certificates.py

```diff
@@ -264,7 +264,7 @@ def _on_certificate_revoked(self, event: CertificateRevokedEvent):
 
 # Increment this PATCH version before using `charmcraft publish-lib` or reset
 # to 0 if you are raising the major API version
-LIBPATCH = 4
+LIBPATCH = 3
```

Comment:
We are decrementing the patch level?

Reply:
Yeah it seems like it but no, the actual version on charmhub is 1.2. I'm not sure why it was 4 here.

Reply:
Ok, thanks!